Library/Homebrew/commands.rb

```diff
@@ -129,11 +132,39 @@ def external_commands
     Tap.cmd_directories.flat_map do |path|
       find_commands(path).select(&:executable?)
                          .map(&method(:basename_without_extension))
-                         .map { |p| p.to_s.sub(/^brew(cask)?-/, '\1 ').strip }
+                         .map { |p| p.to_s.delete_prefix("brew-").strip }
     end.map(&:to_s)
        .sort
   end
 
+  def cask_commands(aliases: false)
+    cmds = cask_internal_commands
+    cmds += cask_internal_command_aliases if aliases
+    cmds += cask_external_commands
+    cmds
+  end
+
+  def cask_internal_commands
+    Cask::Cmd.commands
+  end
+
+  def cask_internal_command_aliases
+    Cask::Cmd.aliases.keys
+  end
+
+  def cask_external_commands
+    PATH.new(Tap.cmd_directories, ENV["HOMEBREW_PATH"]).flat_map do |search_path|
+      find_commands(search_path).map do |possible_command|
+        path = possible_command.to_path
+        command_name = path.match(/brewcask-(.*)\.rb/) { |data| data[1].delete_suffix(".rb") }
+        if command_name.blank? && possible_command.executable?
+          command_name = path.match(/brewcask-(.*)/) { |data| data[1] }
```

Comment:
Can `second` be used instead of `[1]` here?

Reply:
Tried it and `.second` isn't defined on `MatchData`

Reply:
My bad, sorry!